### PR TITLE
grass.jupyter: add ipyleaflet backend

### PR DESCRIPTION
**Pull request by petrasovaa**
This PR adds ipyleaflet backend to jupyter interactiveMap, so one can choose that or folium. Right now it's not doing anything else, the functionality is the same. The hope is ipyleaflet can be used for more interactive tasks, such as querying map, drawing etc.

### CODE REVIEW

File: doc/notebooks/jupyter_tutorial.ipynb
Comment (chaedri) [inline doc/notebooks/jupyter_tutorial.ipynb:1]: To make the folium and ipyleaflet maps the same size, we could put this in for the ipyleaflet cell:
```
import ipyleaflet 
from ipywidgets import Layout

# Create map
m = ipyleaflet.Map(center=[35.761168,-78.668271], layout=Layout(width='600px', height='400px'))
```

File: python/grass/jupyter/interactivemap.py [hunk 1]
```diff
@@ -10,18 +10,30 @@
 #            License (>=v2). Read the file COPYING that comes with GRASS
 #            for details.
```
Comment (chaedri) [inline python/grass/jupyter/interactivemap.py:12]: @petrasovaa You should probably have your name in the authors sections too.

**Review by chaedri — APPROVED** [review]
Looks great! Thank you. I will try to work on some advanced ipyleaflet examples/integration.

File: doc/notebooks/jupyter_tutorial.ipynb
Comment (petrasovaa) [reply doc/notebooks/jupyter_tutorial.ipynb:1]: I removed the folium.Figure from the example instead, so the maps are now both stretched out and the examples are simpler.

**Review by wenzeslaus — APPROVED** [review]

**Pull request merged**